app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -148,6 +149,15 @@ export const FixedSessionsMapCtrl = (
       }
 
       setupTimeRangeFilter(elmApp, $scope.sessions, callback,  params.get('data').timeFrom, params.get('data').timeTo);
+
+      new Clipboard('.copy-link');
+      elmApp.ports.requestCurrentUrl.subscribe(() => {
+        const longUrl = window.location.href
+
+        $http.get("api/short_url", { params: { longUrl: longUrl }})
```

Comment:
Please use `fetch` in place of `$http`. Since we are moving away from angular better not to use its helpers

Reply:
`{ longUrl: longUrl }` == { longUrl }`